dependencies.json

```
@@ -1,13 +1,13 @@
 {
-    "version_warpx": "25.11",
+    "version_warpx": "25.12",
     "version_amrex": "25.11",
     "version_pyamrex": "25.11",
     "version_picsar": "25.06",
     "version_pybind11": "v3.0.1",
     "version_picmi": "0.34.0",
-    "commit_amrex": "c4fcda1fb2ea238c3e6fa273dc9cfcde54b4cc51",
-    "commit_pyamrex": "c7c6e455a585ca43853009fb8df6fdce9e0a65cd",
-    "commit_picsar": "0c329e66010267662a82219f7de7abbd231463f4",
+    "commit_amrex": "25.11",
+    "commit_pyamrex": "25.11",
```

Commit this once AMReX 25.12 and pyAMReX 25.12 are released:

```suggestion
    "commit_amrex": "25.12",
    "commit_pyamrex": "25.12",
```